test-suite/js/src/test/require-jdk8/org/scalajs/testsuite/javalib/time/DurationTest.scala

```diff
@@ -622,22 +622,28 @@ object DurationTest extends JasmineTest with ExpectExceptions {
 
     it("should respond to `addTo`") {
       val t = LocalTime.NOON
+      val d = LocalDate.MIN
```

Comment:
Consider adding a couple of these tests on some of the more obscure `Temporal`s.